src/media-gallery/media-gallery.js

```diff
@@ -123,16 +135,38 @@ component.template = `
             <span class="media-gallery__screen-reader-only">Remove</span>
         </button>
     </div>
+    <div class="media-list__media-container" ng-if="vm.view === vm.viewPreferences.LIST"
+         ng-repeat="media in vm.media">
```

Comment:
Does this also need the limitTo thats in the Tile view?

Reply:
I wasn't sure it made sense for the list view because the list view is, in my mind, the ability to see bulk amounts of attachments.